common/expression/tests/it/main.rs

```diff
@@ -334,6 +491,30 @@ fn builtin_functions() -> FunctionRegistry {
         |lhs, rhs| lhs + rhs,
     );
 
+    registry.register_2_arg::<NumberType<i32>, NumberType<i32>, NumberType<i32>, _, _>(
+        "minus",
+        FunctionProperty::default(),
+        |lhs, rhs| {
+            Some(IntDomain {
+                min: lhs.min.checked_add(rhs.min).unwrap_or(i32::MAX as i64),
+                max: lhs.max.checked_add(rhs.max).unwrap_or(i32::MAX as i64),
```

Comment:
The domain calculation seems not right.

Reply:
it is same as the **plus** function🤔

Reply:
For example, given `a` in `10..=15` and `b` in `-5..=5`, `minus(a, b)` can be in `5..=20`, because `10-5=5` and `15-(-5) = 20`.

Reply:
In general, `min=lhs.min-rhs.max` and `max=lhs.max-rhs.min`.

Reply:
fixed it